### PR TITLE
E2E test for examples/storm

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

File: test/e2e/examples.go [hunk 1]
```diff
@@ -167,6 +167,51 @@ var _ = Describe("Examples e2e", func() {
 			})
 		})
 	})
+
+	Describe("[Skipped][Example]Storm", func() {
```
Comment (jayunit100) [inline test/e2e/examples.go]: IIRC `[Skipped]` is a syntax specifically for density.go, right?  Or is it broader now.

**Comment by bprashanth** [comment]
Please provide some background about the test (either a bug, pr description or block comments around the test itself)

File: test/e2e/examples.go (hunk 1 above)
Comment (mwielgus) [inline test/e2e/examples.go]: Updated comment int e2e_test.go

**Comment by mwielgus** [comment]
@bprashanth Umbrella bug: #10013
Added more By(), the test follows the scenario described in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/storm/README.md.

File: examples/storm/storm-worker-controller.json [hunk 2]
```diff
@@ -41,12 +41,7 @@
                 "hostPort": 6703,
                 "containerPort": 6703
               }
-            ],
-            "resources": {
-              "limits": {
-                "cpu": "200m"
-              }
-            }
+            ]
```
Comment (gmarek) [inline examples/storm/storm-worker-controller.json]: Is this necessary? I'd rather have limits specified everywhere to give users good examples;)

**Comment by gmarek** [comment]
LGTM short of limits thingy.

File: examples/storm/storm-worker-controller.json (hunk 2 above)
Comment (mwielgus) [inline examples/storm/storm-worker-controller.json]: reverted

**Comment by mwielgus** [comment]
Limits thingy fixed.

**Comment by gmarek** [comment]
@wojtek-t

**Pull request merged**